common/build.gradle

```diff
@@ -169,6 +169,7 @@ dependencies {
     implementation "androidx.credentials:credentials:$rootProject.ext.AndroidCredentialsVersion"
     //needed for credentials support from play services, for devices running
```

Comment:
How would this behave on non-GMS devices?

Reply:
(if you're asking about line 172, the legacy API), my best guess is that we're going to get an error response back from the API, which the client logic will turn into an exception to be returned back to the server side.

Reply:
We ship brokers on nGMS devices, so my question is around Passkey/LegacyFIDO flow in particular. Want to make sure we're not breaking those devices.

Reply:
I'll ask Intune and some other internal folks about this. I do think some nGMS customers are also interested in this feature, so you're right, this is a pretty relevant question.